3-extra/1-choose-your-own-adventure.js

```diff
@@ -47,6 +48,43 @@ let game = {
   },
 
   move: function (direction) {
+    if (this.currentRoom === "hall" && direction === "north") {
```

Comment:
you can group the if statements something like this:
```
if (this.currentRoom === "hall") {
  if (direction === "north) {
....
  } else if (direction === "west") {
  } 
  .....
} else if (this.currentRoom === "classroom") {
....
```

Reply:
do you know solution for this test I could not finsh it?